tests/IVIMmodels/unit_tests/reduce_output_size.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+
+df = pd.read_csv('test_output.csv')
```

Comment:
This is particularly inefficient because it loads everything and operates on it all in memory at once. This is a big file so it might be worth either fixing this now or opening an issue to do this later.

An optimization would be to not use pandas and use the simpler csv and operate line by line. Python can still handle the zipping or it could be the linux utility.

Reply:
Done.